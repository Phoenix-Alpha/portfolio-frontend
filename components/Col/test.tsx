import { shallow } from 'enzyme'
import toJSON from 'enzyme-to-json'

import { Col } from './index'

describe('<Col />', () => {
  it('should renders children', () => {
    const text = 'test'

    const wrapper = shallow(<Col>{text}</Col>)

    expect(wrapper.text()).toMatch(text)

    expect(toJSON(wrapper)).toMatchSnapshot()
  })
})
