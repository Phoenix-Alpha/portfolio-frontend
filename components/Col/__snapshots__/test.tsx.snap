// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Col /> should renders children 1`] = `
<div
  className="px-4 w-full"
>
  test
</div>
`;
